docs/tutorials/prometheus.ipynb

```diff
@@ -0,0 +1,471 @@
+        "\n",
+        "[CoreDNS](https://github.com/coredns/coredns) is a DNS server with a focus on service discovery, and is widely deployed as a part of the [Kubernetes](https://kubernetes.io) cluster. For that reason it is often closely monitoring by devops operations.\n",
+        "\n",
+        "This tutorial is an example that could be used by devops looking for automation in their operations through machine learning."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "MUXex9ctTuDB"
+      },
+      "source": [
+        "## Setup and usage"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "upgCc3gXybsA",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Install required tensorflow-io package, and restart runtime"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "uUDYyMZRfkX4",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "try:\n",
+        "  %tensorflow_version 2.x\n",
+        "except Exception:\n",
+        "  pass\n",
+        "!pip install tensorflow-io-nightly"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "yZmI7l_GykcW",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Install and setup CoreDNS and Prometheus\n",
+        "\n",
+        "For demo purposes, a CoreDNS server locally with port `9053` open to receive DNS queries and port `9153` (defult) open to expose metrics for scraping. A simple Corefile configuration of CoreDNS is as follows and is available to be [downloaded](https://github.com/tensorflow/io/blob/master/docs/tutorials/prometheus/Corefile):\n",
+        "```\n",
+        ".:9053 {\n",
+        "  prometheus\n",
+        "  whoami\n",
+        "}\n",
+        "```\n",
+        "\n",
+        "More details about installation could be found on CoreDNS's [documentation](https://coredns.io).\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "YUj0878jPyz7",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "!curl -s -OL https://github.com/coredns/coredns/releases/download/v1.6.7/coredns_1.6.7_linux_amd64.tgz\n",
+        "!tar -xzf coredns_1.6.7_linux_amd64.tgz\n",
+        "\n",
+        "!curl -s -OL https://raw.githubusercontent.com/tensorflow/io/master/docs/tutorials/prometheus/Corefile\n",
+        "\n",
+        "!cat Corefile\n",
+        "\n",
+        "# Run `./coredns` as a background process.\n",
+        "# IPython doesn't recognize `&` in inline bash cells.\n",
+        "get_ipython().system_raw('./coredns &')"
```

Comment:
Use separate code cells for shell commands and python. We have styles that make this clearer when imported into the website.